lib/matplotlib/cm.py

```diff
@@ -301,19 +297,15 @@ def _get_cmap(name=None, lut=None):
 get_cmap = _api.deprecated(
     '3.6',
```

Comment:
```suggestion
    '3.7',
```

This one seems to have escaped the bumping to 3.7 that the other ones got